config/default.yml

```
@@ -1564,7 +1564,7 @@ Layout/TrailingWhitespace:
   StyleGuide: '#no-trailing-whitespace'
   Enabled: true
   VersionAdded: '0.49'
-  VersionChanged: '1.0'
+  VersionChanged: '<<next>>'
```

This update is unnecessary as the default configuration remains unchanged.

```suggestion
  VersionChanged: '1.0'
```